src/components/ContextualMenu/ContextualMenu.scss

```diff
@@ -87,4 +87,18 @@ $ContextualMenu-iconWidth: 24px;
   @include right(8px);
   width: 16px;
   text-align: center;
+}
+
+button.ms-ContextualMenu-link,
+button.ms-ContextualMenu-link:first-child,
+button.ms-ContextualMenu-link:last-child {
+    height: 40px;
+}
+
+.ms-ContextualMenu.ms-FocusZone {
+  position: relative;
+  line-height: 0;
+  border: none;
+  box-shadow: 0px 0px 0px 0px transparent;
```

Comment:
i'm still not totally sure whats up here; we shouldn't need to apply a box-shadow here; where is it coming from?